pkg/util/util.go

```diff
@@ -299,33 +294,7 @@ func ClusterOperatorStatusProgressingCondition(s *samplev1.Config, degradedState
 // ClusterNeedsCreds checks the conditions that drive whether the operator complains about
 // needing credentials to import RHEL content
 func ClusterNeedsCreds(s *samplev1.Config) bool {
-	if strings.TrimSpace(s.Spec.SamplesRegistry) != "" &&
-		strings.TrimSpace(s.Spec.SamplesRegistry) != "registry.redhat.io" {
-		return false
-	}
-
-	if s.Spec.ManagementState == operatorv1.Removed ||
-		s.Spec.ManagementState == operatorv1.Unmanaged {
-		return false
-	}
-	if s.Status.Conditions == nil {
-		return true
-	}
-
-	// some timing paths can lead to only the  config valid condition existing,
-	// so explicitly check it the import creds condition is even there yet
-	foundImportCred := false
-	for _, rc := range s.Status.Conditions {
-		if rc.Type == samplev1.ImportCredentialsExist {
-			foundImportCred = true
-			break
-		}
-	}
-	if !foundImportCred {
-		return true
-	}
-
-	return ConditionFalse(s, samplev1.ImportCredentialsExist)
+	return false
```

Comment:
could just delete this method as well ... will also do when we have e2e's passing